### DISCUSSION TITLE
Any plans to serve SAP UI5 wc from CDN?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [UI5/webcomponents#2573] [Ideas]
I would love to be able to load SAP UI5 into my app from a modern CDN such as  [skypack.dev](https://www.skypack.dev/)! This would eliminate the need for packagers and builders and plays really nice with HTTP2 and the upcoming [import maps](https://wicg.github.io/import-maps/)

**Comment by hrstoyanov** [comment]
oops .. spoke too soon. It looks like UI5 wc is automatically converted and  available, for example - see this:
**https://www.skypack.dev/view/@ui5/webcomponents**
Not sure how usable it is though... Can you please test?

**Reply by jithinkumarvs** [reply]
I'm just new to Web Component. Is there any way we can run this web component based project on Web IDE Full Stack and deploy to cloud foundry?

**Reply by pskelin** [reply]
Hi @jithinkumarvs ,
you can check this blog post for an explanation of why we don't offer a CDN version of ui5 webcomponents.

It comes with a few downsides and it is not so difficult to create your own bundle. I would recommend to follow the steps in the blog
https://blogs.sap.com/2021/05/28/getting-started-with-ui5-web-components-in-2021/

UI5 web components getting started guide also contains the same instructions, do a `npm run build` after you import the needed components and submit the `dist` folder inside your Web IDE project.

**Comment by jithinkumarvs** [comment]
Hi Peter Skelin,

Thanks for the immediate response with detailed information. It was helpful.

Regards,
Jithin

On Fri, Jun 18, 2021 at 9:11 AM Peter Skelin ***@***.***>
wrote:

> Hi @jithinkumarvs <https://github.com/jithinkumarvs> ,
> you can check this blog post for an explanation of why we don't offer a
> CDN version of ui5 webcomponents.
>
> It comes with a few downsides and it is not so difficult to create your
> own bundle. I would recommend to follow the steps in the blog
>
> https://blogs.sap.com/2021/05/28/getting-started-with-ui5-web-components-in-2021/
>
> UI5 web components getting started guide also contains the same
> instructions, do a npm run build after you import the needed components
> and submit the dist folder inside your Web IDE project.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SAP/ui5-webcomponents/discussions/2573#discussioncomment-886360>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADMJWOSXIAB7P4GOK2ZJUDTTTLPSPANCNFSM4UXKUO3Q>
> .
>


-- 


Thanks & Regards,

JITHIN KUMAR VS



:: jithin.for2011@ ***@***.***>gmail.com

:: 889-256-7383 (India)

:: https://in.linkedin.com/in/jithinkumarvs
